WordPress/build.gradle

```diff
@@ -58,17 +58,20 @@ android {
     productFlavors {
         vanilla { // used for release and beta
             dimension "buildType"
+            buildConfigField "boolean", "EXPERIMENTAL_CATEGORY_AVAILABLE", "false"
```

Comment:
Hmm, perhaps `EXPERIMENTAL_CATEGORY_AVAILABLE` is not very appropriate for the particular feature. Perhaps this is a copy/paste mishap?

Reply:
Hey @hypest , On my mind was that we can generally have an experimental category in the App Settings which isn't dedicated to Enable Gutenberg Editor switch only, but also for the future experimental stuff. But if you think that we don't need that kind of approach right now, I am fine with that?

Reply:
Oh I see. What we traditionally have done with these feature flags is to have them as specific as possible so, I'd say let's make this one only relevant to gutenberg-mobile and the inclusion of the block editor in particular.

So, my suggestion would be: "OFFER_GUTENBERG" or similar.

Reply:
Agreed with @hypest let's make the naming super obvious for everyone, `OFFER_GUTENBERG` seems spotted on